source/_integrations/owntracks.markdown

```diff
@@ -24,7 +24,9 @@ To configure OwnTracks, you must set it up via the integrations panel in the con
 
 [Install the OwnTracks application for Android.](https://play.google.com/store/apps/details?id=org.owntracks.android)
 
-In the OwnTracks app, open sidebar and click on preferences, then on connection. Change the following settings:
+[Install the NextTracks application for Android.](https://codeberg.org/nexttracks/android/releases) (Soon available from [F-Droid](https://f-droid.org/))
```

Comment:
The way it is currently documented, makes it look like you need to install both.